Gemfile

```diff
@@ -35,11 +35,12 @@ group :pam_authentication, optional: true do
 end
 
 gem 'net-ldap', '~> 0.18'
-gem 'omniauth-cas', '~> 2.0'
-gem 'omniauth-saml', '~> 1.10'
+
+gem 'omniauth-cas', github: 'stanhu/omniauth-cas', branch: 'sh-update-omniauth2'
```

Comment:
I think I would use an explicit commit hash instead of a branch here, as it's not something we control.